lib/Configuration.js

```diff
@@ -7,6 +7,8 @@ import semver from 'semver';
 
 import FileUtils from './FileUtils';
 import JsModule from './JsModule';
+import meteor from './environments/meteor';
+import node from './environments/node';
```

Comment:
I think it would be better if these files were called `meteorEnvironment` and `nodeEnvironment` for maximum readability.